public/keymaps/c/chavdai40_default.json

```
@@ -0,0 +1,19 @@
+{
+    "keyboard": "chavdai40",
+    "keymap": "default",
+    "layout": "LAYOUT_42key",
```

Please change the layout macro to `LAYOUT_44key`, and add source commit hash for version tracking.

```suggestion
    "commit": "245797f6b998fb4f52df61e38151c80b05615740",
    "layout": "LAYOUT_44key",
```